src/CMakeLists.txt

```diff
@@ -211,13 +212,12 @@ configure_file(
 install(FILES
     "${PROJECT_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/AppImageKitConfig.cmake"
     "${PROJECT_BINARY_DIR}/AppImageKitConfigVersion.cmake"
-    DESTINATION "/lib/CMake/AppImage"
-    COMPONENT dev
+    DESTINATION "lib/cmake/AppImageKit"
```

Comment:
You mentioned the previous path was problematic cross-platform wise. Did you test this on different platforms? If yes, which ones?

Reply:
Only ubuntu/debian by the moment. What we could do is use a variable here and let the distro packers to use the path they want.

Reply:
If it works on Debian/Ubuntu, then it's fine. Other packagers will probably patch it directly, or step up to us to ask about it.